doc/ref_model/chapters/chapter02.md

```diff
@@ -170,8 +176,9 @@ By trying to sort workloads into different categories based on the requirements
 # 2.3 Cloud Infrastructure Profiles
 
 Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
-- **Basic**: for Workloads that can tolerate resource over-subscription and variable latency.
-- **Network Intensive**: for Workloads that require predictable computing performance, high network throughput and low network latency.
+- **Basic**: For Workloads that can tolerate resource over-subscription and variable latency.
```

Comment:
Any category you see for compute intensive or special purpose compute like GPUs etc. ?

Reply:
@msidana , the initial profiles include compute-intensive but we removed it as there is no use cases for it that time